src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.js

```diff
@@ -110,9 +112,10 @@ const SplitForm = ({ incident, attachments, onHandleCancel, onHandleSubmit }) =>
       create,
       update,
     });
-  }, [incident.id, isVisible, onHandleSubmit]);
+  }, [incident, isVisible, onHandleSubmit]);
 
   useEffect(() => {
+    if (!incident) return;
```

Comment:
Can you add a test case for this condition?

Reply:
done!